db/upgrade.php

```diff
@@ -341,5 +343,31 @@ function xmldb_cms_upgrade($oldversion) {
         upgrade_mod_savepoint(true, 2023112100, 'cms');
     }
 
+    if ($oldversion < 2023120101) {
+        // Add adhoc task to update contextid in customfield records.
+        manager::queue_adhoc_task(new update_customfield_context());
+        upgrade_mod_savepoint(true, 2023120101, 'cms');
+    }
+
+    if ($oldversion < 2023120102) {
+        $dbman = $DB->get_manager();
+        // Conditionally launch add field valuetrust.
+        if ($dbman->field_exists('customfield_data', 'valuetrust')) {
+            $sql = "SELECT mcd.id mcdid
+                      FROM {customfield_data} mcd
+                      JOIN {customfield_field} mcf ON mcf.id = mcd.fieldid
+                      JOIN {customfield_category} mcc ON mcc.id = mcf.categoryid
+                     WHERE mcc.component = 'mod_cms' AND mcd.valuetrust = 0";
```

Comment:
@TomoTsuyuki we need to filter records a bit more and set  update only records where `area = 'cmsfield'`

Reply:
Actually I think it's ok.